自動更新機能を実装

avaklee opened the pull request:
# WHAT

自動更新機能を実装, 前回の章で抜けていたイメージのif文追加

# WHY

他のユーザーが投稿した時に、現ユーザーがリロードしなくても更新されるようにするため

avaklee committed:
自動更新機能を実装

# WHAT

自動更新機能を実装, 前回の章で抜けていたイメージのif文追加

# WHY

他のユーザーが投稿した時に、現ユーザーがリロードしなくても更新されるようにするため

seoky0 commented on app/assets/javascripts/message.js:
```diff
@@ -1,5 +1,6 @@
-$(function() {
-  function buildHTML(message) {
+$(function(){
+  // $(function msg(){
```
不要なコメントアウトは消しましょう

seoky0 commented on app/assets/javascripts/message.js:
```diff
@@ -10,8 +11,13 @@ $(function() {
-      + message_image
       + '</div>'
+      if (message.image.url ==! ""){
+      document.write(
```
document.writeは必要でしょうか？

seoky0 commented on app/views/messages/create.json.jbuilder:
```diff
@@ -1,3 +1,4 @@
+
```
不要な空行なので消しましょう

avaklee committed:
自動更新の実装（修正）

# WHAT

1. GEM turbolink off
2. HTML のimage　読み込みのコードを追加
3.  Link をつけてあげた

# WHY

1.ajaxが呼ばれてなかった
2.文字列になっていたので、追加
3.今までやっていなかったので
